apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js

```diff
@@ -194,46 +498,85 @@ const IdeaForm = (
                     >
                       <img alt="submit_image" src={popoverSvg} />
                     </atoms.ToolTip>
-                  </atoms.Typography>
-                  <atoms.Box flexDirection="column" alignItems="flex-start">
-                    <atoms.Radio
-                      onChange={(e, label) =>
-                        setFieldValue('involveLevel', 'highly')
+                  </GroupLabel>
+
+                  <RadioButton>
+                    <RadioInput
+                      type="radio"
+                      id="highly"
+                      name="involveLevel"
+                      value="highly"
+                      checked={values.involveLevel === 'highly'}
+                      onChange={(e) =>
+                        setFieldValue('involveLevel', e.target.value)
                       }
-                      label='I want to "own" this idea from beginning to end'
+                      onBlur={handleBlur}
                     />
-                    <atoms.Radio
-                      onChange={(e, label) =>
-                        setFieldValue('involveLevel', 'medium')
+                    <RadioLabel htmlFor="highly">
+                      <strong>Level 1 - Rocket</strong>
```

Comment:
@veratao where is this text/label from? We never implemented that

Reply:
This is from Figma